ext/opentelemetry-ext-flask/tests/test_automatic.py

```diff
@@ -0,0 +1,98 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import NOTSET, WARNING, disable
+from unittest import main
+
+# This is used instead of from flask import Flask, request because if not then
+# FlaskInstrumentor().instrument() would need to be called before importing
+# Flask. This is just an intrinsic limitation due the fact that we are testing
+# the instrumentor in a way that mimics how it would be called with the
+# opentelemetry-auto-instrumentation command. This does not mean that the
+# instrumentor should be used in this way in end user applications. For those
+# cases, FlaskInstrumentor.instrument(app=app) should be used.
```

Comment:
```suggestion
# cases, FlaskInstrumentor.instrument_app(app) should be used.
```

?

Reply:
That comment does not apply anymore, removed.